### DISCUSSION TITLE
unable to re-apply for student

### DISCUSSION THREAD

**Discussion by oscie57** [community/community#57184] [GitHub Education]
### Hi everyone,

I previously had the GitHub student pack, and I noticed the other day that I was missing some of my benefits. Upon opening the GitHub education page, I noticed I couldn't access the benefits page, and that I have to re-apply. This makes sense, as it does say that I may occasionally have to prove I go there, and it has been 2 years since I last did.

i used to have the GitHub student pack, and i noticed around a month ago that i couldn't access my benefits anymore. i tried re-applying as it seemed i didnt have it anymore, but then i got this message:

> You have an outstanding discount request for @oscie57. We will get back to you soon.

i didnt get an email or anything about this. i made a support ticket about a month ago and havent had a response, and i had made a discussion here previously. im hoping i get some help this time around.

**Comment by samus-aran** [comment]
Hi @oscie57 👋 

Occasionally it is necessary for us to ask you to reverify your current academic status in order to continue receiving your GitHub Education benefits. We may require this when seemingly-legitimate applicants have submitted academic affiliation proof which has been discovered to be used by bad actors as well.
We apologize for any inconvenience this may cause.
To reverify, return to the GitHub Education benefits page( https://education.github.com/benefits ) and click "submit your information again."

Usually that message means you are currently mid-way in an application submission. You can cancel your current application or wait until it  is deleted automatically.

Thank you for your understanding.
